lib/chat_api_web/router.ex

```diff
@@ -46,12 +46,14 @@ defmodule ChatApiWeb.Router do
 
     get("/slack/oauth", SlackController, :oauth)
     get("/slack/authorization", SlackController, :authorization)
+    put("/widget_settings/create_or_update", WidgetSettingController, :create_or_update)
 
     resources("/user_invitations", UserInvitationController, except: [:new, :edit])
     resources("/accounts", AccountController, only: [:update, :delete])
     resources("/messages", MessageController, except: [:new, :edit])
     resources("/conversations", ConversationController, except: [:new, :edit, :create])
     resources("/customers", CustomerController, except: [:new, :edit, :create])
+    resources "/widget_settings", WidgetSettingController, only: [:index, :delete]
```

Comment:
if we're not using any of these endpoints, we can probably just remove this (we should be doing this more in general, but might as well start here)